src/components/TaskPopup.jsx

```diff
@@ -69,17 +69,25 @@ const TaskPopup = ({ task, onClose }) => {
           <div className={styles.ExtraBtnContainer}>
             <div className={styles.Extra}>
               <h4 className={styles.ExtraTitle}>Add to card</h4>
-              <button className={styles.ExtraBtn}>Deadline</button>
-              <button className={styles.ExtraBtn}>Do Date</button>
-              <button className={styles.ExtraBtn}>Member</button>
-              <button className={styles.ExtraBtn}>Extra</button>
-              <button className={styles.ExtraBtn}>Extra</button>
-              <button className={styles.ExtraBtn}>Extra</button>
-              <button className={styles.ExtraBtn}>Extra</button>
-              <button className={styles.ExtraBtn}>Extra</button>
-              <button className={styles.ExtraBtn}>Extra</button>
-              <button className={styles.ExtraBtn}>Extra</button>
-              <button className={styles.ExtraBtn}>Extra</button>
+              <button className={`${styles.ExtraBtn} ${styles.DeadlineBtn}`}>
+                Deadline
+              </button>
+              <div className={styles.FlexBtn}>
+                <button className={`${styles.ExtraBtn} ${styles.DoDateBtn}`}>
+                  Do Date
+                </button>
+              </div>
```

Comment:
this button is wrapped in a div and makes style look weird:

![image](https://github.com/EllenSandqvist/kanban_collab/assets/58448612/a1fce6fe-f9cf-47cc-9f82-0164132bbbd6)